main_test.go

```diff
@@ -172,7 +197,10 @@ func newTest(t *testing.T) test {
 					Name:       "pod-with-finalizer",
 					Namespace:  "default",
 					Finalizers: []string{"yes.com/hello"},
-					Labels:     map[string]string{hasFinalizerLabel: "true"},
+					Labels: map[string]string{
+						orgRouteAll + hasFinalizerLabel:        "true", // We don't expect any reconciliation for orgRouteTest on this resource
```

Comment:
I'd argue that the comment needs to be on it's own line, as it doesn't belong to the `orgRouteAll` line :) 

But I also don't think we need those comments at all. I think the label speaks for itself, and if not, maybe we should make it so :)

Reply:
> I'd argue that the comment needs to be on it's own line,

It was, but linter betrayed my manual line breaks 👎 
But you are right, we can just drop the comment